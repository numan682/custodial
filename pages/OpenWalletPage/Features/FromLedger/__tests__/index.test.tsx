import * as React from 'react'
import { render } from '@testing-library/react'
import { FromLedger } from '..'

jest.mock('react-redux', () => ({
  useSelector: jest.fn(),
  useDispatch: jest.fn(),
}))

describe('<FromLedger />', () => {
  it('should render component', () => {
    const { container } = render(<FromLedger />)

    expect(container).toMatchSnapshot()
  })
})
