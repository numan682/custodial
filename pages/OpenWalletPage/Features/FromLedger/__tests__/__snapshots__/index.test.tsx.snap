// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FromLedger /> should render component 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  box-sizing: border-box;
  max-width: 100%;
  margin: 12px;
  background-color: #FFFFFF;
  color: #444444;
  border: solid 1px background-front-border;
  min-width: 0;
  min-height: 0;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  padding: 24px;
  border-radius: 5px;
}

.c3 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  box-sizing: border-box;
  max-width: 100%;
  margin-top: 24px;
  min-width: 0;
  min-height: 0;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
}

.c4 {
  display: inline-block;
  box-sizing: border-box;
  cursor: pointer;
  font: inherit;
  -webkit-text-decoration: none;
  text-decoration: none;
  margin: 0;
  background: transparent;
  overflow: visible;
  text-transform: none;
  border: 2px solid #7D4CDB;
  border-radius: 18px;
  color: #444444;
  padding: 4px 22px;
  font-size: 18px;
  line-height: 24px;
  background-color: #7D4CDB;
  color: #f8f8f8;
  border-radius: 18px;
  -webkit-transition-property: color,background-color,border-color,box-shadow;
  transition-property: color,background-color,border-color,box-shadow;
  -webkit-transition-duration: 0.1s;
  transition-duration: 0.1s;
  -webkit-transition-timing-function: ease-in-out;
  transition-timing-function: ease-in-out;
}

.c4:hover {
  box-shadow: 0px 0px 0px 2px #7D4CDB;
}

.c4:focus {
  outline: none;
  box-shadow: 0 0 2px 2px #6FFFB0;
}

.c4:focus > circle,
.c4:focus > ellipse,
.c4:focus > line,
.c4:focus > path,
.c4:focus > polygon,
.c4:focus > polyline,
.c4:focus > rect {
  outline: none;
  box-shadow: 0 0 2px 2px #6FFFB0;
}

.c4:focus::-moz-focus-inner {
  border: 0;
}

.c4:focus:not(:focus-visible) {
  outline: none;
  box-shadow: none;
}

.c4:focus:not(:focus-visible) > circle,.c4:focus:not(:focus-visible) > ellipse,
.c4:focus:not(:focus-visible) > line,.c4:focus:not(:focus-visible) > path,
.c4:focus:not(:focus-visible) > polygon,.c4:focus:not(:focus-visible) > polyline,
.c4:focus:not(:focus-visible) > rect {
  outline: none;
  box-shadow: none;
}

.c4:focus:not(:focus-visible)::-moz-focus-inner {
  border: 0;
}

.c1 {
  margin-top: 0px;
  font-size: 34px;
  line-height: 40px;
  max-width: 816px;
  font-weight: 600;
  overflow-wrap: break-word;
}

.c2 {
  margin: 0;
  font-size: 26px;
  line-height: 32px;
  max-width: 624px;
  font-weight: 600;
  overflow-wrap: break-word;
}

@media only screen and (max-width:768px) {
  .c0 {
    margin: 6px;
  }
}

@media only screen and (max-width:768px) {
  .c0 {
    border: solid 1px background-front-border;
  }
}

@media only screen and (max-width:768px) {
  .c0 {
    padding: 12px;
  }
}

@media only screen and (max-width:768px) {
  .c3 {
    margin-top: 12px;
  }
}

@media only screen and (max-width:768px) {
  .c1 {
    margin-top: 0px;
  }
}

@media only screen and (max-width:768px) {
  .c1 {
    font-size: 26px;
    line-height: 32px;
    max-width: 624px;
  }
}

@media only screen and (max-width:768px) {
  .c2 {
    margin: 0;
  }
}

@media only screen and (max-width:768px) {
  .c2 {
    font-size: 18px;
    line-height: 24px;
    max-width: 432px;
  }
}

<div>
  <div
    class="c0"
  >
    <h1
      class="c1"
    >
      openWallet.ledger.header
    </h1>
    <h3
      class="c2"
    >
      ledger.instructionSteps.header
    </h3>
    <ol>
      <li>
        ledger.instructionSteps.connectLedger
      </li>
      <li>
        ledger.instructionSteps.closeLedgerLive
      </li>
      <li>
        ledger.instructionSteps.openOasisApp
      </li>
    </ol>
    <div
      class="c3"
    >
      <button
        class="c4"
        type="submit"
      >
        openWallet.importAccounts.selectWallets
      </button>
    </div>
  </div>
</div>
`;
